### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by Roxanne718**
#4

### CODE REVIEW

File: src/puppet-lark.ts [hunk 1]
```diff
@@ -272,22 +296,28 @@ class PuppetLark extends Puppet {
   }
 
   protected contactRawPayload (contactId: string): Promise<any> {
+    console.info(contactId)
```
Comment (huan) [inline src/puppet-lark.ts:299]: change to:

```ts
log.verbose('PuppetLark', 'contactRawPayload(%s)', contactId)
```

**Review by huan — CHANGES_REQUESTED** [review]
Your fixes look great.

Please follow my reviews and change the logging messages to align to our standard message format, and then we will be able to merge it and move on.

Thank you very much!

**Comment by Roxanne718** [comment]
> Your fixes look great.
> 
> Please follow my reviews and change the logging messages to align to our standard message format, and then we will be able to merge it and move on.
> 
> Thank you very much!

I've modified the code, please check it again. Thanks！

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
This is a huge PR and it fixed most of the problems from the code base earlier.

Now all linting have been passed and it's ready for us to move forward.

Thank you @Roxanne718 for your effort to improve the quality of our code base, appreciate it!

**Pull request merged**